scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -430,18 +464,20 @@ do
   elif [ "${1}" = "add-plugin" ]
   then
     shift
-    doSetup silent
```

Comment:
if you remove this and eclipse is not installed, then `doAddPlugin` will fail with `command not found: eclipse`.
IMHO this should not be removed.

Reply:
readded doSetup silent